Automattic/newspack-theme
# Add second custom color option

### All Submissions:


### Changes proposed in this Pull Request:

The secondary colour has only been added to the content links so far (which alone, needed a lot of variables to be changed). We still need to finalize exactly where it should be applied otherwise in the theme, and added there in a future PR. 

This PR also:

* Cleans up comments in color-patterns.php, to reference the colour being applied rather than what it's being applied to. 
* Removes theme's Dark Grey and Light Grey from the editor colour palette; we will want to have some default colours that are always there, but we'll need to figure out what they should be once the default colour palettes are finalized. As is, with a default secondary grey colour, you end up with a lot of different greys. 
* Removes the `postMessage` transport from the Customizer colour options, and the JavaScript preview. Unfortunately, incorporating multiple colours makes previewing like this more complex -- it's something to revisit in the future, once the colour behaviour is finalized, but is not a breaking issue (the Customizer will just do a full refresh when the colour is changed, rather than previewing without). 

See #24.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Confirm that there is a mid-range and dark grey added to the editor as colour options.
3. In the Customizer, confirm that there is an option for a second custom colour. 
4. Change the colour; confirm it changes links in the content body, and the colour options in the editor. 

### Other information:
